kda/graph_utils.py

```diff
@@ -214,10 +214,7 @@ def get_ccw_cycle(cycle, order):
         input cycle. This pair of nodes is used to determine which direction is
         CCW.
     """
-    CCW = _is_ccw(cycle, order[0], order[1])
-    if CCW == True:
+    if _is_ccw(cycle, order[0], order[1]):
         return cycle
-    elif not CCW:
-        return cycle[::-1]
     else:
-        raise CycleError("Direction of cycle {} could not be determined.".format(cycle))
+        return cycle[::-1]
```

Comment:
Now that I think about it, we might need to keep this error in the event the node indices in `order` do not exist in `cycle`. But I think that should be handled differently, probably before `is_ccw()` is called. Something like the following should do the trick:

```python
if not all(i in cycle for i in order):
    raise CycleError(f"Input node indices {order} do not exist in cycle {cycle}.")
``` 

This is still better than before, because the previous error message made a claim that doesn't really make sense -- If the indices in `order` exist in the cycle, the cycle is either CCW or CW.